anvil-rstudio-bioconductor-devel/VERSION

```diff
@@ -0,0 +1 @@
+0.0.1
```

Comment:
Is the proposal to make this `3.13.0` to sync the `bioconductor` and `anvil-docker` versions?

(Or should it be `3.13.0-devel` if the release hasn't been cut yet? Not sure.)

Reply:
Hi @rtitle 

I want to replace the 0.0.1 completely, so that the version number is in sync with bioconductor_docker i.e 3.13.0 ....

I don't want to call it `3.13.0-devel` because in the future 3.13.x will become the release.

Reply:
OK, I'm okay with just using bioconductor versions in this repo